src/classes/com/sun/tdk/jcov/util/Utils.java

```diff
@@ -1189,8 +1190,17 @@ public static void checkFile(File file, String description, CheckOptions... opts
                     }
                     break;
                 case FILE_CANWRITE:
```

Comment:
I assume you have reviewed all usages of FILE_CANWRITE, because by this change you do change the meaning of it.

Reply:
Yes, right now the CANWRITE is wider that just writing into existing file is allowed. Here it gives write permissions to non-existing file in existing directory that has write permissions

Reply:
My question was whether you checked that there are no places in JCov code where CAN_WRITE is used to also make sure that the file exists.

Reply:
Sure, I've checked all placed (at least I believe in this strongly)

Reply:
👍